spring-aot/src/main/java/org/springframework/nativex/type/TypeSystem.java

```diff
@@ -565,6 +565,12 @@ private void scanArchive(File f) {
 				// TODO resources?
 			}
 		} catch (IOException ioe) {
+			if(ioe instanceof NoSuchFileException) {
```

Comment:
I put in a change to allow for empty classpath elements on a scan (the regular indexing was already coping with it) - so this should not be needed any more I think.